app/models/dialog.rb

```diff
@@ -89,6 +89,7 @@ def validate_field_data
   end
 
   def load_values_into_fields(values)
+    values = values.with_indifferent_access
```

Comment:
Would this change be required in `initialize_with_given_values` below where we are doing the same key lookup?

Reply:
In theory, it wouldn't hurt I guess. In practice that path is only being used by reconfigure and uses the contents of `options[:dialog]` within a service.